### DISCUSSION TITLE
css and js doesn't work

### DISCUSSION THREAD

**Discussion by Nurzhhh** [Laravel-Backpack/community-forum#1126] [Q&A (Help)]
Hello! After updating to version 6.x, part of css and js not working, for example:
![Screenshot 2024-08-21 at 14 27 09](https://github.com/user-attachments/assets/5d1df64b-120a-4ff3-acff-d99cd345ca94)

**Comment by Nurzhhh** [comment]
Also daterangepicker did not work, so I added in ui.php 
'scripts' => [
        'https://unpkg.com/moment@2.29.4/min/moment-with-locales.min.js',
        'https://unpkg.com/bootstrap-daterangepicker@3.1.0/daterangepicker.js'
    ],
    globally, after that it was solved

**Comment by karandatwani92** [comment]
Hey @Nurzhhh 

It seems you were upgrading from an old version to v6.
The latest release doesn't have any such issue. Can you recheck:
- Step 16 & 17 from the upgrade [guide](https://backpackforlaravel.com/docs/6.x/upgrade-guide#css-js-assets).
- `php artisan basset:clear`
- `php artisan basset:cache`

**Reply by karandatwani92** [reply]
More info on basset is available [here](https://github.com/Laravel-Backpack/basset).

**Reply by Nurzhhh** [reply]
Thanks, I'll try
